index.html

```diff
@@ -35,7 +35,12 @@
       <header class="d-flex font-inter">
```

Comment:
Please add all the unnecessary files and folders like `.vscode` to `.gitignore`.